src/test/common.node.ts

```diff
@@ -322,21 +323,15 @@ export async function captureScreenShot(contextOrFileName: string | Mocha.Contex
     if (!isCI) {
         return;
     }
-    const fullTestNameHash =
-        typeof contextOrFileName === 'string'
-            ? ''
-            : hashjs
-                  .sha256()
-                  .update(contextOrFileName.currentTest?.fullTitle() || '')
-                  .digest('hex');
     const testTitle = typeof contextOrFileName === 'string' ? '' : contextOrFileName.currentTest?.title || '';
     const fileNamePrefix =
-        typeof contextOrFileName === 'string' ? contextOrFileName : `${testTitle}_${fullTestNameHash}`;
-    const name = `${fileNamePrefix}_${uuid()}`.replace(/[\W]+/g, '_');
+        typeof contextOrFileName === 'string' ? contextOrFileName : `${testTitle}_${screenShotIndex}`;
+    const name = fileNamePrefix.replace(/[\W]+/g, '_');
```

Comment:
Can we keep the hash and the screenshot index and just remove the `uuid`.
I think that will reduce the length.
Optionally we could also reduce the length of the test title by taking just the first 100 characters `testTitle.substring(0, 100)`

The hash is used in another place, so we can determine the screenshots that belong to a particular test.
Basiscally we now generate ipynb files with the test failures and screenshots of the errors, i demnoe'd this last week, probably should share this again.

If  you go into the CI job and look for artifacts named 'TestResultNotebook...ipynb` that will contain the failed tests, with test logs for that test and the corresponding screenshots as well.
I find that very useful, else I have to parse the entire log file manually & then look for screenshots manually, this way the ipynb has everything in one place.

Reply:
I wonder if that notebook works on windows at the moment. I should try that and see.

Reply:
Hmm, I don't see a failure with screenshots.

Reply:
Did you remove the separate screenshots now that we have a notebook? 

On this run, I would have expected there to be screenshots:
https://github.com/microsoft/vscode-jupyter/actions/runs/2659693681